### PR TITLE
DOC: misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../pandas-whitelist.txt`
Where whitelists consists of:
```
ans
behaviour
doubleclick
indicies
initialise
initialised
initialising
nd
resetted
splitted
thru
valu
```

### CODE REVIEW

File: pandas/tests/frame/test_repr_info.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def test_info_memory_usage(self):
         res = buf.getvalue().splitlines()
         assert "memory usage: " in res[-1]
 
-        # do not display memory usage cas
+        # do not display memory usage case
```
Comment (luzpaz) [inline pandas/tests/frame/test_repr_info.py]: please review

File: pandas/tests/sparse/frame/test_frame.py [hunk 2]
```diff
@@ -247,10 +247,10 @@ def test_constructor_preserve_attr(self):
     def test_constructor_nan_dataframe(self):
         # GH 10079
         trains = np.arange(100)
-        tresholds = [10, 20, 30, 40, 50, 60]
-        tuples = [(i, j) for i in trains for j in tresholds]
+        thresholds = [10, 20, 30, 40, 50, 60]
+        tuples = [(i, j) for i in trains for j in thresholds]
```
Comment (luzpaz) [inline pandas/tests/sparse/frame/test_frame.py]: source code change, please review

File: pandas/tests/sparse/frame/test_frame.py [hunk 3]
```diff
@@ -247,10 +247,10 @@ def test_constructor_preserve_attr(self):
...
         index = pd.MultiIndex.from_tuples(tuples,
-                                          names=['trains', 'tresholds'])
+                                          names=['trains', 'thresholds'])
```
Comment (luzpaz) [inline pandas/tests/sparse/frame/test_frame.py]: ditto

**Comment by TomAugspurger** [comment]
Thank you!

**Pull request merged**